### ISSUE TITLE
tls-basic.sh test failure: gnutls error -43: Error in the certificate

### ISSUE THREAD

**Issue by Whissi** [rsyslog/librelp#112]
```
============================================================
Test: ./tls-basic.sh
============================================================
Starting Receiver...
got receive pid 128604
startup wait 0
program started up, pidfile receive.pid contains 128604
Receiver running
Send Message...
send: authentication error 'certificate validation failed', object ''
send: error 'TLS handshake failed [gnutls error -43: Error in the certificate.]', object 'conn to srvr 127.0.0.1:30514'
failure in: relpCltConnect(pRelpClt, protFamily, port, target)
receive: error 'TLS handshake failed [gnutls error -54: Error in the pull function.]', object 'lstn 30514: conn to clt 1
27.0.0.1/127.0.0.1'
receive: error 'TLS record write failed [gnutls error -10: The specified session has been invalidated for some reason.]', object 'lstn 30514: conn to clt 127.0.0.1/127.0.0.1'
receiver stopped

FAIL: expected message not found. Expected:
testmessage

librelp.out.log actually is:
FAIL tls-basic.sh (exit status: 1)
```

**Comment by Whissi** [comment]
Seems to be fixed in git master...

**Comment by Whissi** [comment]
Ah OK. Tests are failing because certificate used for testing isn't valid anymore. Was fixed via 8491c8a0bcbb89c03875f128ed29baf9c05c4e79.
